types/selenium-webdriver/.eslintrc.json

```diff
@@ -4,6 +4,7 @@
         "@definitelytyped/no-unnecessary-generics": "off",
         "@typescript-eslint/ban-types": "off",
         "@typescript-eslint/no-empty-interface": "off",
-        "@typescript-eslint/naming-convention": "off"
+        "@typescript-eslint/naming-convention": "off",
+        "@typescript-eslint/array-type": "off"
```

Comment:
Are you disabling this because of a bug, or for some other reason?

Reply:
Removed this line. Initially, my first approach aimed to maintain consistency by using T[] instead of Array[]. On second thought, it makes sense to use Array[] for union type.